src/test/test.js

```diff
@@ -34,3 +34,560 @@ describe('UI routes', () => {
       });
   });
 });
+
+
+describe('/Post create political party', () => {
```

Comment:
Separate test into different files according to what you are testing for. So all test for party should be in a file called `party.test.js` or just `party.js` and so on. That way your code is cleaner and failing tests can easily be identified instead of having a bulk load of tests in one file